Rawl_B502/genelach_ríg_n_alban.trig

```diff
@@ -367,8 +367,7 @@
     irishRel:genName "Forggo";
     irishRel:nomName "Forggu";
     rel:childOf <#Feradaig>;
-    owl:sameAs <http://example.com/LL/rig_ulad.trig#Forgo>;
-    owl:sameAs <http://example.com/LL/rig_alban.trig#Morga>.
+    owl:sameAs <http://example.com/LL/rig_ulad.trig#Forgo>.
```

Comment:
What makes you say `owl:sameAs <http://example.com/LL/rig_alban.trig#Morga>` is incorrect? They seem to have the same ancestry.

Reply:
Because I get something like the attached image and it does not look right to me.  Maybe you should investigate it more because I am out of ideas as to how to proceed :(

![Screenshot_2020-07-11_15-14-56](https://user-images.githubusercontent.com/350395/87226086-6e7f6500-c389-11ea-8554-ec58c9fb687a.png)

Reply:
I don't see anything illogical here? Forgg is the son of Feradach and has two children, Maine and Roshin.

Reply:
Sorry, on just saw your edit: "F" > "M" is not a common spelling variation and, if their descendants are also different, it does sound best to keep "Morga" and "Forg(a)" separate.